src/science/misc/cable_carbon.F90

```diff
@@ -150,8 +155,16 @@ SUBROUTINE carbon_pl(dels, soil, ssnow, veg, canopy, bgc)
     wbav = MAX( 0.01, wbav )  ! EAK Jan2011
 
     ! drought stress
-    coef_drght = EXP( 5.*( MIN( 1., MAX( 1., wbav**( 2 - soil%ibp2 ) - 1.) /    &
-         ( soil%swilt**( 2 - soil%ibp2 ) - 1. ) ) - 1. ) )
+    CampbellExp         = 2.0 - soil%ibp2
```

Comment:
I wouldn't use `CampbellExp` as the working variable name - unless it is exactly the same (and same science intent) as the Campbell parts of the soil moisture science.

Otherwise the break up of the equation is good (possibly over the top)